src/api/config/encoder.rs

```diff
@@ -213,7 +221,7 @@ impl EncoderConfig {
     // has the property that the scaled distortion of a 2Nx2N block is always
     // equal to the sum of the scaled distortions of the NxN sub-blocks it's
     // made of, this is a necessary property to be able to do RDO between
-    // multiple partition sizes properly. Unfortunately, when tx domain
+    // multiple partition sizes properly. Unfortunately, when tx domains
```

Comment:
What's this change for? At first glance it seems like singular domain is grammatically correct.